### PR TITLE
Create bloomsbury-academic.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/83409/style-request-bloomsbury-academic#latest

### CODE REVIEW

File: bloomsbury-academic.csl [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+          <label form="short" suffix=" "/>
+          <name and="text" delimiter-precedes-last="never" initialize-with=". "/>
+        </names>
+      </if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" name-as-sort-order="first"/>
+      <label form="short" text-case="capitalize-first" strip-periods="true" prefix=" (" suffix=")"/>
```
Comment (adam3smith) [inline bloomsbury-academic.csl:49]:
```suggestion
      <label form="short" prefix=", "/>
```

File: bloomsbury-academic.csl [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+      </substitute>
+    </names>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="report">
+        <text variable="title" quotes="true"/>
+      </if>
+      <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <text variable="title" font-style="italic"/>
```
Comment (adam3smith) [inline bloomsbury-academic.csl:80]: I'm seeing title case throughout in the guide

File: bloomsbury-academic.csl [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+        <names variable="editor translator" delimiter=", ">
+          <label form="short" suffix=" "/>
+          <name and="text" delimiter-precedes-last="never" initialize-with=". "/>
+        </names>
+      </if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" name-as-sort-order="first"/>
```
Comment (adam3smith) [inline bloomsbury-academic.csl:48]:
```suggestion
      <name and="text" delimiter-precedes-last="never" name-as-sort-order="first" initialize-with=". "/>
```

**Review by adam3smith — CHANGES_REQUESTED** [review]
Some changes, one question

File: bloomsbury-academic.csl (hunk 2 above)
Comment (POBrien333) [reply bloomsbury-academic.csl]: Thanks. I've fixed that.

File: bloomsbury-academic.csl (hunk 3 above)
Comment (adam3smith) [reply bloomsbury-academic.csl:48]: Was I wrong here or did you just miss this?

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**